src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -547,6 +560,14 @@ private string UFormatDateString(DateTime dateTime)
             return StringUtil.Format(sb.ToString(), dateTime);
         }
 
+        /// <summary>
+        /// Converts unix time into DateTime.
+        /// </summary>
+        private static DateTime UnixTimeToDateTime(long unixTime)
```

Comment:
The method is used only once and can be removed.

Reply:
Great point, updated! 😁